### DISCUSSION TITLE
https://vuepress-theme-hope.github.io/v2/comment/zh/demo.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#550] [Announcements]
# https://vuepress-theme-hope.github.io/v2/comment/zh/demo.html

为了让你更好的了解如何使用插件，我们使用 vuepress-plugin-comment2 和 @vuepress/theme-default 制作了一个案例。 源代码 (https://github.com/vuepress-theme-hope/vuepress-theme-hope/tree/main/demo/comment2/); 预览 (h...

https://vuepress-theme-hope.github.io/v2/comment/zh/demo.html

**Comment by lincheng77** [comment]
shiyishi
